### DISCUSSION TITLE
How to get code for the internally created pages.

### DISCUSSION THREAD

**Discussion by RohanNayaks** [oqtane/oqtane.framework#1371] [Q&A]
I created a internal module using file management , but I couldn't find the page code in the solution explorer.

Please help.

**Comment by leigh-pointer** [comment]
@RohanNayaks what version of the framework are you using?

**Reply by RohanNayaks** [reply]
Hey @leigh-pointer  , 2.1.0

**Reply by leigh-pointer** [reply]
When creating a module, source code is found outside of the framework solution.  You will find it on your file system on the same path as the framework solution folder.

Modules are loaded into the framework at runtime.
